pkgs/development/ocaml-modules/cstruct/ppx.nix

```
@@ -13,20 +12,8 @@ buildDunePackage {
 
   minimumOCamlVersion = "4.07";
```

```suggestion
  minimalOCamlVersion = "4.08";
```

Build of `ocaml-ng.ocamlPackages_4_07.ppx_cstruct` fails with:

>   File "ppx/ppx_cstruct.ml", line 347, characters 27-36:
  Error: This expression has type 'a option
         but an expression was expected of type string

---

Done.

---

This was also a suggestion to fix the spelling…

---

Oh right, completely forgot that change happened. Maybe we should remove those treewide before 21.11 and make the old spelling a warning? Either way fixed.